core-services/secrets/hypershift-clusterbot/hypershift-clusterbot.yaml

```diff
@@ -0,0 +1,52 @@
+# this is the Namespace in which your Secret will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation Secrets for HyperShift cluster-bot support
+    openshift.io/display-name: HyperShift Cluster Bot
+  name: hypershift-clusterbot
+---
+# the Group of people who should be able to manage this Secret
+kind: Group
+apiVersion: v1
+metadata:
+  name: hypershift-clusterbot-admins
+users:
+  # these names are GitHub usernames
+  - csrwng
+  - enxebre
+  - ironcladlou
+  - sjenning
+---
+# this adds the admins to the project.
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hypershift-clusterbot-viewer-binding
```

Comment:
I'd remove binding from the name since it's implicit in the kind

Reply:
Fixed